modules/core/src/main/java/org/apache/ignite/internal/pagemem/wal/record/FilteredRecord.java

```diff
@@ -16,12 +16,12 @@
 */
 package org.apache.ignite.internal.pagemem.wal.record;
 
-import org.apache.ignite.internal.processors.cache.persistence.wal.AbstractWalRecordsIterator;
+import org.apache.ignite.internal.processors.cache.persistence.wal.AbstractFileWalRecordsIterator;
 
 /**
  * Special type of WAL record. Shouldn't be stored in file.
  * Returned by deserializer if next record is not matched by filter. Automatically handled by
- * {@link AbstractWalRecordsIterator}.
+ * {@link AbstractFileWalRecordsIterator}.
```

Comment:
It is also handled by `ByteBufferWalIterator` now. Let's remove this sentence at all.